src/Workspaces/Core/Portable/Notification/GlobalOperationNotificationService.cs

```diff
@@ -90,7 +90,7 @@ private void Done(GlobalOperationRegistration registration)
             if (!Environment.HasShutdownStarted)
             {
                 Contract.ThrowIfFalse(_registrations.Count == 0);
-                Contract.ThrowIfFalse(_operations.Count == 0);
+                Contract.ThrowIfFalse(_operations.Count == 0, $"Non-disposed operations: {string.Join(", ", _operations)}");
```

Comment:
note: this should always be empty.  and even if it's not, this is a interpolated string handler.  so it's no alloc unless it fires.